packages/storybook/src/nimble/table/table-fit-rows-height-matrix.stories.ts

```diff
@@ -29,6 +29,12 @@ const groupingStates = [
 ] as const;
 type GroupingState = (typeof groupingStates)[number];
 
+const minColumnWidthStates = [
+    ['Small Minimum', 100],
+    ['Large Minimum', 500]
```

Comment:
Any spot to capture the scrollbar intention, maybe in the name?

```suggestion
    ['Large Minimum (show horizontal scrollbar)', 500]
```